packages/frontend/src/components/dialogs/Transports/index.tsx

```diff
@@ -77,11 +77,10 @@ export default function TransportsDialog(
       'bcc_self'
     )
     if (multiDeviceMode === '1') {
-      openAlertDialog({
+      await openAlertDialog({
         message:
-          'It is not possible to add transports when multi-device mode is enabled.',
+          'Note if using multi-device:\nbefore changing or adding transports make sure all other devices have at least version 2.33.0 installed. Otherwise they will run out of sync.',
```

Comment:
IDK if "they will run out of sync" is clear. Run out of sync in what way? You said that the sent messages will not appear on old versions. I think that "they will run out of sync" is too weak of a wording.

Reply:
Synced in what way? What's supposed to be in sync and what is not? Encrytion / decryption keys? App themes? Accounts? Photos?

Also this doesn't say how hard it is to get them back in sync.